### PR TITLE
Add Ubuntu 22.04 support for development snapshots

### PR DESCRIPTION
**Pull request by shahmishal**
Support Ubuntu 22.04 
### Motivation:

Add latest Ubuntu version

### Modifications:

Add links to Ubuntu 22.04 download page.

### Result:

Screen Shot 2022-06-03 at 12 01 51 AM

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

File: download/_linux.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Note that nothing prevents Swift from being ported to other Linux distributions
 
 #### Requirements
 
-* Ubuntu 18.04 or 20.04 (64-bit)
-* CentOS 7, 8
```
Comment (0xTim) [inline download/_linux.md:11]: CentOS 8 is still listed elsewhere - are we dropping support for it?

**Review by 0xTim — APPROVED** [review]
LGTM other than a CentOS query

**Comment by shahmishal** [comment]
@swift-ci test

**Pull request merged**